### DISCUSSION TITLE
The Gif show the ur with pick and place box

### DISCUSSION THREAD

**Discussion by ashley1223** [jr-robotics/robo-gym#36] [Q&A]
Hi all, thank you for the framework.
May I ask any chance can put the ur robot pick and place example? The one gif show. Thank you!

**Comment by matteolucchi** [comment]
Hi @ashley1223 !

I am sorry but I don't understand, which GIF are you referring to?

**Reply by ashley1223** [reply]
Hi Matteolucchi,
Thank you for reply soon! The gif(video) in robo-gym repository , Readme part, one is ur robot pick the red box.

**Reply by ashley1223** [reply]
![81711381-e73e6f80-9473-11ea-880e-1b0ff50e15ff](https://user-images.githubusercontent.com/36742328/124146919-2370cd00-da86-11eb-9c4a-726c6f8d78cc.gif)

**Reply by matteolucchi** [reply]
Hi, this is not a pick and place task but only and Effector Positioning task, you can find more info about it [here](https://github.com/jr-robotics/robo-gym/blob/master/docs/environments.md#end-effector-positioning)

**Reply by ashley1223** [reply]
Thank you very much!

**Reply by matteolucchi** [reply]
No problem :)

**Reply by TheQuyen97ht** [reply]
I'm on my way starting a new task "Pick and Place" based on robo-gym. Will the repository get updated with this task, or can you give me some advice to update based on End-efector reaching task that has been already availble ?
